cypress/integration/login.spec.ts

```diff
@@ -0,0 +1,23 @@
+const loginScreenTestIds = {
+  loginHeader: 'login-header',
+  loginEmail: 'login-form__input-email',
+  loginPassWord: 'login-form__input-password',
+  loginSubmit: 'login-form__button-submit',
+};
+describe('Login page', () => {
+  it('render the login page with the right element', () => {
```

Comment:
As this is `it` - third-person singular verb, then we should change the verb:
```suggestion
  it('renders the login page with the right element', () => {
```

Please check other examples as well 🙏